cmd/krel/cmd/announce_send.go

```diff
@@ -169,8 +169,20 @@ func runAnnounce(opts *sendAnnounceOptions, announceRootOpts *announceOptions, r
 
 	logrus.Info("Sending mail")
 	subject := fmt.Sprintf("Kubernetes %s is live!", tag)
-	if err := m.Send(content, subject); err != nil {
-		return errors.Wrap(err, "unable to send mail")
+
+	yes := true
+
+	if rootOpts.nomock {
+		_, yes, err = util.Ask(fmt.Sprintf("%s (Y/n)", "Send email?"), "y:Y:yes|n:N:no|y", 10)
```

Comment:
Tiny nit: do we really need `fmt.Sprintf` here? I think something like this would work as well:

```suggestion
		_, yes, err = util.Ask("Send email? (Y/n)", "Send email?"), "y:Y:yes|n:N:no|y", 10)
```

Also, isn't the default value `no`, at least in case of `nomock`? I'm asking this, because in this case, `Y` should be lowercase, and instead `N` should be uppercase.

```suggestion
		_, yes, err = util.Ask(fmt.Sprintf("%s (y/N)", "Send email?"), "y:Y:yes|n:N:no|y", 10)
```

Reply:
Agree, just one nit of the nit. To make N the default, change it like this:
```suggestion
		_, yes, err = util.Ask("Send email? (Y/n)", "y:Y:yes|n:N:no|N", 10)
```

Reply:
Done! Thank you @xmudrii and @puerco for your reviews!